core/themes/basis/css/component/node.css

```diff
@@ -0,0 +1,25 @@
+/**
+ * @file
+ * Styling for node content.
+ */
+
+.node {
+  position: relative;
+}
+
+/* Display "Draft" watermark on unpublished content. */
+/* See basis_preprocess_node(). */
+.node[data-status]:before {
+  content: attr(data-status);
+  position: absolute;
+  z-index: 5;
+  width: 100%;
```

Comment:
Should have `display: block;` or `inline-block`, I'm surprised width is working at all, may not in other browsers.